docs: Add frontmatter to `standards/*` pages (#23095)
MystenLabs/sui: docs/content/standards/closed-loop-token/token-policy.mdx

---
title: Token Policy

> description: A `TokenPolicy` is a shared object that defines a token's allowed actions and conditions. `TokenPolicies` are often used by wallets and other financial services.

Reviewer: ```suggestion
description: A `TokenPolicy` is a shared object that defines a token's allowed actions and conditions. Wallets and other financial services often use `TokenPolicy` objects.
```
